roles/ocp-route-redirect/tasks/redirect.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- k8s_facts:
+    api_version: route.openshift.io/v1
+    kind: Route
+    name: "{{ route_redirect_name }}"
+    namespace: "{{ route_redirect_namespace }}"
+  register: route
+
+- set_fact:
+    route: "http://{{ (route.resources | first).spec.host }}"
+
+- uri:
+    url: "{{ route }}"
+    validate_certs: "{{ route_redirect_validate_certs | default(false) }}"
```

Comment:
This is related to the previous comment, does this need to be configurable if the protocol is always `http`?
If the protocol is intended to also be configurable I would suggest setting the default to `true` and make it a conscious decision of the user to not validate the certs.

Reply:
Parameterized the protocol and have defaulted the validation to true 👍